resources/js/ziggy.js

```diff
@@ -23,7 +23,7 @@ const Ziggy = {
         about: { uri: 'about', methods: ['GET', 'HEAD'] },
         search: { uri: 'search', methods: ['GET', 'HEAD'] },
         handleRedirect: { uri: 'redirect', methods: ['GET', 'HEAD'] },
-        'event.index': { uri: 'event', methods: ['GET', 'HEAD'] },
+        event: { uri: 'event', methods: ['GET', 'HEAD'] },
```

Comment:
Let's not edit this file, it's generated on prod push, so everything gets overwritten, should probably .gitignore it.